firestore-tos-extension/extension.yaml

```diff
@@ -0,0 +1,145 @@
+name: acknowledgements-api
+version: 0.0.1
+specVersion: v1beta
+
+displayName: Ackowledgement Management Extension
+description: Provides an out of the box acknowledgement management extension for Firestore 
+
+license: Apache-2.0
+
+sourceUrl: https://github.com/invertase/firebase-extensions/tree/main/extensions/storage-image-processing-api
+releaseNotesUrl: https://github.com/invertase/firebase-extensions/tree/main/extensions/storage-image-processing-api/CHANGELOG.md
+
+author:
+  authorName: Invertase
+  url: https://invertase.io
+
+contributors:
+  - authorName: Invertase
+    email: oss@invertase.io
+    url: https://github.com/invertase
+
+billingRequired: true
+
+roles:
+  - role: firebaseauth.admin
+    reason: >-
+      Allows the extension to set custom claims for users.
+
+  - role: datastore.user
+    reason: Allows this extension to access Cloud Firestore to read and process notices.
+
+resources:
+  - name: acceptNotices
```

Comment:
maybe acceptNotice? because the endpoint doesn't let you accept multiple at once?

Reply:
Naming updated.